pseudo_dojo/dojo/works.py

```diff
@@ -55,7 +55,7 @@ def write_dojo_report(self, report, overwrite_data=False):
         old_report[dojo_trial][dojo_ecut] = report
         try:
             self.pseudo.write_dojo_report(old_report)
-        except (OSError, IOError, TypeError):
+        except None: #(OSError, IOError, TypeError):
```

Comment:
I still have a pull request open, there are some more commits added


--
Université catholique de Louvain
SST/IMCN - Institute of Condensed Matter and Nanosciences (IMCN)

Dr. Michiel J. van Setten

NAPS - Chemin des Étoiles 8 bte L7.03.01
1348 Louvain-la-Neuve (Belgium)
Email: michiel.vansetten@uclouvain.be mjvansetten@gmail.com
Telefon: +32 10 478681
--

2015-08-21 20:13 GMT+02:00 gmatteo <notifications@github.com>:

> In pseudo_dojo/dojo/works.py
> <https://github.com/gmatteo/pseudo_dojo/pull/30#discussion_r37661866>:
>
> > @@ -55,7 +55,7 @@ def write_dojo_report(self, report, overwrite_data=False):
> >          old_report[dojo_trial][dojo_ecut] = report
> >          try:
> >              self.pseudo.write_dojo_report(old_report)
> > -        except (OSError, IOError, TypeError):
> > +        except None: #(OSError, IOError, TypeError):
>
> Please use
>
> try:
>
> except:
>
> if you want to trap all the possible exceptions.
> In python3, one cannot use arbitrary objects wit except, only subclasses
> of BaseException are allowed
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gmatteo/pseudo_dojo/pull/30/files#r37661866>.
>